easypost/billing.py

```diff
@@ -0,0 +1,81 @@
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+)
+
+from easypost.easypost_object import convert_to_easypost_object
+from easypost.error import Error
+from easypost.requestor import (
+    RequestMethod,
+    Requestor,
+)
+from easypost.resource import (
+    CreateResource,
+    Resource,
+)
+
+
+class Billing(CreateResource, Resource):
+    @classmethod
+    def fund_wallet(cls, amount: str, primary_or_secondary: str = "primary", api_key: Optional[str] = None) -> bool:
+        """Fund your EasyPost wallet by charging your primary or secondary payment method on file."""
```

Comment:
Do we not detail parameters in docstrings in Python?

Reply:
No, not in Python yet